src/components/singleComment.jsx

```diff
@@ -1,6 +1,20 @@
-import React from "react";
+import React, { useState } from "react";
+import { useLogin } from "../contexts/login";
+import { deleteComment } from "../utils/api";
+
+const SingleComment = ({ comment, setIsCommentDeleted, isCommentDeleted }) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const { user } = useLogin();
+
+  function handleClick(comment_id) {
+    setIsLoading(true)
+    deleteComment(comment_id)
+      .then(() => {
+        setIsCommentDeleted(!isCommentDeleted);
+        setIsLoading(false)
+      })
+  }
```

Comment:
Great consideration of loading state! 
It would also be good to implement `error` state to demonstrate to a user when the request has failed, currently it will sit in an infinite loading state 🔁